Firebase/DynamicLinks/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # v3.2.0
+- Add support for creating and receiving dynamic links from custom domains. This feature is not yet available for public consumption. (#1962, #2017, #2078)
```

Comment:
This needs to change and better reflect what's available now - introduced a new `componentsWithLink:domainURIPrefix:` and deprecated the existing `componentsWithLink:domain:`. There should be no mention of custom domains since it's not available for consumption yet.

Reply:
Done.